### DISCUSSION TITLE
Customizing Theme CSS

### DISCUSSION THREAD

**Discussion by rroarings** [arifszn/gitprofile#46] [Q&A] [answered]
Hi Arif,

Is it possible to customize a specific theme? Such as changing colors, font options, etc.

Thanks.

**Comment by arifszn** [comment, marked as answer]
Expect an update today about adding a custom theme option. Meanwhile, explore this [doc](https://daisyui.com/docs/themes/).

**Reply by rroarings** [reply]
Sweet news, thanks.

**Reply by arifszn** [reply]
Custom theme option is added. 🎉🎉

https://github.com/arifszn/gitprofile#themes
